common/components/chrome/MainHeader.jsx

```diff
@@ -53,437 +41,160 @@ class MainHeader extends React.Component<{||}, State > {
-
-              <a href={url.section(Section.AboutUs)}>
-                <div className={'SubHeader-drawerDiv'} >
-                  About Us
-                </div>
-              </a>
-              <Divider />
-
-              <a href={url.section(Section.Press)}>
-                <div className={'SubHeader-drawerDiv'} >
-                  News
-                </div>
-              </a>
-              <Divider />
-
-              {window.BLOG_URL && <React.Fragment><a href={window.BLOG_URL}>
-                <div className={'SubHeader-drawerDiv'} >
-                  Blog
-                </div>
-              </a>
-              <Divider />
-              </React.Fragment>}
-
-              {this._renderHamburgerNavigationLinks()}
-
-              <Divider />
-              {
-                !CurrentUser.isLoggedIn() &&
-                <React.Fragment>
-                  <div className={'SubHeader-drawerDiv'} onClick={this._onLogInClick}>
-                    <a href="" > Log In </a>
-                  </div>
-                  <Divider />
-                </React.Fragment>
-              }
-              {
-                CurrentUser.isLoggedIn() &&
-                <a href="/logout/">
-                  <div className={'SubHeader-drawerDiv'}>
-                    Logout
-                  </div>
-                </a>
-              }
-            </div>
-          </div>
-        </Drawer>
+        <Button className="MainHeader-showdesktop MainHeader-donate-link" variant="link" href={url.section(Section.Donate)}>Donate</Button>
+        <Button className="MainHeader-showdesktop MainHeader-login-button" variant="outline-primary" href={url.section(Section.LogIn, url.getPreviousPageArg())}>Log In</Button>
       </React.Fragment>
-
     )
   }
 
-  _renderHamburgerNavigationLinks(): $ReadOnlyArray<React$Node> {
-    //TODO: refactor NavigationLinks to define link behavior on a per-link basis
-    return NavigationLinks.list().map((link) => {
-      return (
-        <React.Fragment key={link.url}>
-          <a key={link.url} href={link.url} rel="noopener noreferrer" onClick={NavigationLinks.logClick.bind(this, link)}>
-            <div className={'SubHeader-drawerDiv'}>
-              {link.name}
-            </div>
-          </a>
-          <Divider />
-        </React.Fragment>
-      );
-    });
+//TODO: Refactor these to reduce duplication
+//TODO: Allow multiple arguments for url.section to handle url.getPreviousPageArg() - options object?
+  _renderNavLink(section, text, classes = "") {
+    const urlArgs = url.arguments(url.section(section));
+    if (urlArgs.section === this.state.activeSection) {
+      classes += " MainHeader-active";
+    };
+    return <Nav.Link className={classes} href={url.section(section)}>{text}</Nav.Link>
   }
-
-  _toggleSlider(open: boolean): void {
-    this.setState({ slider: open })
+  _renderNavDropdownItem(section, text, classes = "") {
+    const urlArgs = url.arguments(url.section(section));
```

Comment:
This code snippet is duplicated with _renderNavLink.  Perhaps we can wrap it into a helper method?